recipe/meta.yaml

```diff
@@ -2,20 +2,26 @@ package:
     name: mingwpy
     version: 0.1.0
 
-source:                                                                                                # [win]
+source:
```

Comment:
This is being brought back in this PR ( https://github.com/conda-forge/mingwpy-feedstock/pull/8 ).